Sources/AardvarkMailUI/ARKEmailBugReporter.m

```diff
@@ -493,17 +493,9 @@ - (void)_dismissEmailComposeWindow;
     
     [self.mailComposeViewController endAppearanceTransition];
     
-    static BOOL iOS9OrLater;
-    static dispatch_once_t onceToken;
-    dispatch_once(&onceToken, ^{
-        iOS9OrLater = [[UIDevice currentDevice].systemVersion compare:@"9.0" options:NSNumericSearch] != NSOrderedAscending;
-    });
-    
     // Work around an iOS 9 bug where we don't get UIWindowDidBecomeKeyNotification when the mail compose view controller dismisses.
```

Comment:
While it might be good to see if this bug still exists, let's make it a bit more clear why we're doing this:
```suggestion
    // Work around a bug introduced in iOS 9 where we don't get UIWindowDidBecomeKeyNotification when the mail compose view controller dismisses.
```

Reply:
Yeah, good call